force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -594,8 +594,8 @@
         <fullName>Only_Showing_Future_Sessions</fullName>
         <language>en_US</language>
         <protected>true</protected>
-        <shortDescription>Only showing sessions on or after today&apos;s date</shortDescription>
-        <value>Only showing sessions after today&apos;s date</value>
+        <shortDescription>Only showing sessions on or after today&apos;s date cannot remove existing sessions.</shortDescription>
```

Comment:
@jencrabtree - Took a stab at it to give you an idea of what we are looking for. Thanks
<img width="2056" alt="Screen Shot 2022-07-15 at 10 19 36 AM" src="https://user-images.githubusercontent.com/450473/179255005-b2073020-60e7-47a5-826a-88eae73520f3.png">
!

Reply:
Thank you, @jjbennett! Here's what was approved during review:

Showing all upcoming Sessions. To modify previously added Sessions, go to the Service Sessions list.

Reply:
@jencrabtree, Can we reduce the length of this sentence. We are over the 80 character limit and so we are unable to create scratch orgs and the builds are failing.

Reply:
You can ignore my comment @jencrabtree

Reply:
Yeah it does. We fixed this by changing the short description which is not visible to the end users.